rakelib/rswag.rake

```diff
@@ -23,6 +23,12 @@ def generate_swagger_doc(dev: false, section: nil)
   rswag_to_oas!("modules/appeals_api/app/swagger/appeals_api/v2/swagger#{glob}.json")
 end
 
+def generate_swagger_doc(yaml_path, json_path)
```

Comment:
I don't think we need to update our rake tasks just yet, since the docs controller is doing a yaml-to-json conversion already. We'll worry about updating this when we convert the appeals status docs to rswag.